Orange/widgets/data/owfile.py

```diff
@@ -93,7 +94,12 @@ def __init__(self):
         box.setSizePolicy(Policy.MinimumExpanding, Policy.Fixed)
         self.file_combo.setSizePolicy(Policy.MinimumExpanding, Policy.Fixed)
         self.file_combo.activated[int].connect(self.select_file)
+        self.sheet_combo = sheet_combo = QtGui.QComboBox(box) 
+        sheet_combo.setSizePolicy(Policy.MinimumExpanding, Policy.Fixed) 
+        sheet_combo.setVisible(False)  
```

Comment:
According to [contrib guidelines](https://github.com/biolab/orange3/blob/master/.github/CONTRIBUTING.md#coding-style), prefer Qt properties specified as kwargs to the constructor.